cob_bringup_sim/scripts/move_object.py

```diff
@@ -215,6 +233,11 @@ def parse_options(self):
         parser.add_option("-r", "--radius",
             dest="radius", default=None,
             help="Radius, only used for circular movement. Default: None")
+            
+        parser.add_option("-o", "--object",
```

Comment:
please rename to `stop_objects` (plural) as a list of strings and change code to check against all objects in the list.